CMakeLists.txt

```diff
@@ -162,6 +162,12 @@ install(TARGETS              serac
         )
 
 if (SERAC_ENABLE_CODEVELOP)
+    # This really shouldn't be needed but it keeps getting dropped from axom.
```

Comment:
This should go away once you merge/rebase onto develop

Reply:
I think it's a good idea, but then we'd have to amend all of our copyright statements to
```
// Copyright (c) 2019-2022, Disney, the Marvel Cinematic Universe, Lawrence Livermore National Security, LLC and
// other Serac Project Developers. See the top-level LICENSE file for ...
```
so we'll probably stick with `A`